src/controller/user/Controller.ts

```diff
@@ -87,13 +129,11 @@ class User {
       const Id = req.params.id;
       const toDeleteUser = userData.find((item) => item.id === Id);
       userData.splice(userData.indexOf(toDeleteUser), 1);
-      return res
-        .status(200)
-        .send({
-          message: 'user deleted successfully',
-          data: userData,
-          deleted_User: toDeleteUser,
-        });
+      return res.status(200).send({
+        message: 'user deleted successfully',
+        data: userData,
+        deleted_User: toDeleteUser,
+      });
     } catch (error) {
       return res
```

Comment:
one try-catch is enough for function and throwing error. You need userRepostiory method for CRUD operation not mock one.

Reply:
fixed all bugs now working find.